### PR TITLE
Issue #502: Post-release / post-JOSS README.md update

### LINKED ISSUE #502: post release v0.1.0
## Subject of the issue

- Update logo to remove beta
- Update readme regarding citation

### PR DESCRIPTION
**Pull request by zacbaum**
# Description

* Update logo from beta
* Add pypi badge
* Update contributing with extra information, link to CoC
* Add citation (plain text and bibtex version)

Fixes #502 

## Type of change

What types of changes does your code introduce to DeepReg? _Put an `x` in the boxes that
apply_

- [x] Documentation Update (fix or improvement on the documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR.
If you're unsure about any of them, don't hesitate to ask. We're here to help! This is
simply a reminder of what we are going to look for before merging your code._

- [x] I have
      [installed pre-commit](https://deepreg.readthedocs.io/en/latest/contributing/setup.html)
      using `pre-commit install` and formatted all changed files. If you are not
      certain, run `pre-commit run --all-files`.
- [x] My commits' message styles matches
      [our requested structure](https://deepreg.readthedocs.io/en/latest/contributing/commit.html),
      e.g. `Issue #<issue number>: detailed message`.
- [x] I have updated the
      [change log file](https://github.com/DeepRegNet/DeepReg/blob/main/CHANGELOG.md)
      regarding my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by zacbaum** [comment]
@mathpluscode @NMontanaBrown - please have a look when you can 😄

**Comment by NMontanaBrown** [comment]
> @mathpluscode @NMontanaBrown - please have a look when you can 😄

Is this ready for review or still WIP?

**Comment by zacbaum** [comment]
> > @mathpluscode @NMontanaBrown - please have a look when you can 😄
> 
> Is this ready for review or still WIP?

Ready now - just had to fix pre-commit things (finally working now, @mathpluscode! 😉)

File: README.md [hunk 1]
```diff
@@ -26,17 +27,33 @@ and education in medical image registration using deep learning.**
...
+- [Medical Image Registration Demos using DeepReg](https://deepreg.readthedocs.io/en/latest/demo/introduction.html)
 - [Issue Tracker](https://github.com/DeepRegNet/DeepReg/issues/new/choose)
-- [deepreg.net](http://deepreg.net/)
 
 ## Contributing
 
-Get involved, and help make DeepReg better!
+Get involved, and help make DeepReg better! We want your help - **Really**.
+
+**Being a contributor doesn't just mean writing code.** Equally important to the
```
Comment (NMontanaBrown) [inline README.md:39]: I like this 😄

File: README.md [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+	<img src="https://raw.githubusercontent.com/DeepRegNet/DeepReg/main/docs/asset/deepreg_logo_purple.svg" alt="deepreg_logo" title="DeepReg" width="200" />
 </p>
 
 # DeepReg
 
-[![Build Status](https://travis-ci.com/DeepRegNet/DeepReg.svg?branch=main)](https://travis-ci.com/DeepRegNet/DeepReg)
-[![Documentation Status](https://readthedocs.org/projects/deepreg/badge/?version=latest)](https://deepreg.readthedocs.io/en/latest/?badge=latest)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![Build Status](https://travis-ci.com/DeepRegNet/DeepReg.svg?branch=main)](https://travis-ci.com/DeepRegNet/DeepReg)
```
Comment (NMontanaBrown) [inline README.md:9]: What's the diff here? Is it just the order of badges? I have read over the urls a couple of time but can't spot the difference.

**Review by NMontanaBrown — COMMENTED** [review]
Just curious about the diff in the README for the badges, is it just a change in the order the badges are displayed?

File: README.md (hunk 2 above)
Comment (zacbaum) [reply README.md:9]: Bumped Licence and Black badge to the front, put pypi near the JOSS badge.

Trying to group things (unchanging badges together, status badges together, "external link" badges together)

**Comment by zacbaum** [comment]
> Just curious about the diff in the README for the badges, is it just a change in the order the badges are displayed?

Yep.

Bumped Licence and Black badge to the front, put pypi near the JOSS badge.

Trying to 'group' things (unchanging badges together, status badges together, "external link" badges together)

**Review by NMontanaBrown — APPROVED** [review]

**Review by YipengHu — APPROVED** [review]

**Pull request merged**